app/femr/util/calculations/dateUtils.java

```diff
@@ -42,7 +43,7 @@ public static String getCurrentDateTimeString(){
     }
 
     public static DateTime getCurrentDateTime(){
-        return new DateTime();
+        return new DateTime(DateTimeZone.UTC);
```

Comment:
Hi Kevin,

After the change that we made in the configuration file, I don't see the
use of this line anymore so I believe you are correct!
On Dec 9, 2015 6:33 PM, "Kevin Zurek" <notifications@github.com> wrote:

> In app/femr/util/calculations/dateUtils.java
> <https://github.com/FEMR/femr/pull/460#discussion_r47169804>:
>
> > @@ -42,7 +43,7 @@ public static String getCurrentDateTimeString(){
> >      }
> >
> >      public static DateTime getCurrentDateTime(){
> > -        return new DateTime();
> > +        return new DateTime(DateTimeZone.UTC);
>
> Sam -
>
> Having trouble identifying the significance of this change along with line
> 100 after changing the JDBC connection string, is there one?
>
> -Kevin
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FEMR/femr/pull/460/files#r47169804>.
>